components/navbar/index.tsx

```diff
@@ -5,18 +5,35 @@ export interface NavBarProps {}
 
 const NavBar: FC<NavBarProps> = () => {
   return (
-    <div className="NavBar">
-      <NavItems />
+    <div>
+      <header className="NavBar">
+        <nav>
+          <ul className="NavBar__UL">
+            <NavItems />
```

Comment:
Good job. I would prefer that the parent div had a className of 'NavBar', while header had a className of 'NavBar__Header'